src/Monolog/Formatter/DatadogFormatter.php

```diff
@@ -3,28 +3,16 @@
 namespace sgoettsch\MonologDatadog\Formatter;
 
 use Monolog\Formatter\JsonFormatter;
+use Monolog\Level;
 use Monolog\Logger;
+use Monolog\LogRecord;
 use stdClass;
 
 class DatadogFormatter extends JsonFormatter
 {
-    protected $includeStacktraces = true;
-
-    /**
-     * Map Monolog\Logger levels to Datadog's default status type
-     */
-    private const DATADOG_LEVEL_MAP = [
-        Logger::DEBUG => 'info',
-        Logger::INFO => 'info',
-        Logger::NOTICE => 'warning',
-        Logger::WARNING => 'warning',
-        Logger::ERROR => 'error',
-        Logger::ALERT => 'error',
-        Logger::CRITICAL => 'error',
-        Logger::EMERGENCY => 'error',
-    ];
-
-    public function format(array $record): string
+    protected bool $includeStacktraces = true;
+
+    public function format(LogRecord $record): string
```

Comment:
Monolog changed the array to a class called `LogRecord` that implements `ArrayAccess`. So, it can be easily changed and still used as an array, at least for accessing the values.